tools/coccinelle/README.md

```diff
@@ -0,0 +1,7 @@
+Coccinelle patches
+==================
+
+This collection of coccinelle patches represents some of the broader,
+codebase-wide changes that have been made. If you maintain a fork of
+FRR and find that your codebase needs to be updated to align with
+these changes, the coccinelle tool should help you make that update.
```

Comment:
can you just add the command that the user should run?

Reply:
I've added an example - I think.